core/backend/src/IModelTransformer.ts

```diff
@@ -746,46 +747,50 @@ export class IModelTransformer extends IModelExportHandler {
     return targetElementAspectProps;
   }
 
+  /** The directory where schemas will be exported, a random temporary directory */
+  protected _schemaExportDir: string = path.join(KnownLocations.tmpdir, Guid.createValue());
+
+  /** given a schema identified just by name and the version string, check if it should be imported into the target */
+  private shouldExportSchemaCheck(schemaName: string, versionInSource: string): boolean {
+    const versionInTarget = this.targetDb.querySchemaVersion(schemaName);

```

Comment:
Is shouldExportSchemaCheck called from anywhere else? If not, it could be combined into shouldExportSchema.

Reply:
you're right, the second call path was removed, should be able to remove that now